crates/bevy_render/src/base_bundle.rs

```diff
@@ -0,0 +1,59 @@
+use bevy_ecs::prelude::Bundle;
+use bevy_transform::prelude::{GlobalTransform, Transform};
+
+use crate::view::{ComputedVisibility, Visibility};
+
+/// A [`Bundle`] with the following [`Component`](bevy_ecs::component::Component)s:
+/// * [`Visibility`] and [`ComputedVisibility`], which describe the visibility of an entity
+/// * [`Transform`] and [`GlobalTransform`], which describe the position of an entity
+///
+/// * To show or hide an entity, you should set its [`Visibility`].
+/// * To get the computed visibility of an entity, you should get its [`ComputedVisibility`].
+/// * To place or move an entity, you should set its [`Transform`].
+/// * To get the global position of an entity, you should get its [`GlobalTransform`].
+/// * For hierarchies to work correctly, you must have all four components.
+///   * You may use the [`BaseBundle`] to guarantee this.
```

Comment:
```suggestion
```

I think this is implicitly obvious.

Reply:
as I love writing docs, I mostly copied from
https://github.com/bevyengine/bevy/blob/f531a94370755155accd625c58a06f07d33edcce/crates/bevy_render/src/view/visibility/mod.rs#L96
and
https://github.com/bevyengine/bevy/blob/8eb0440f1ed0f075ed488e7efa36ba0f8bb37cad/crates/bevy_transform/src/lib.rs#L25